index.js

```diff
@@ -91,13 +92,15 @@ module.exports = {
     // No icon pack path defined and an app = legacy case where the app icons must be merged into the 'frost' icon pack
     const isLegacy = !_.has(this, 'app.options.iconPackOptions.path') && !isAddon
 
-    if (isLegacy && fs.existsSync(path.join(this.project.root, 'public/svgs'))) {
-      iconNames['frost'] = _.concat(iconNames['frost'], this.flattenIcons([], '', path.join(this.project.root, 'public/svgs')))
+    const svgPath = path.join(this.project.root, 'publich/svgs')
```

Comment:
The less commonly acknowledged spelling of `public` it's a silent `h` at the end ;)